java/compaction/compaction-status-store/src/test/java/sleeper/compaction/status/store/job/QueryCompactionJobStatusUnfinishedIT.java

```diff
@@ -74,6 +75,7 @@ public void shouldExcludeFinishedCompactionJob() {
         store.jobCreated(job2);
         store.jobStarted(compactionJobStarted(job2, defaultStartTime()).taskId(DEFAULT_TASK_ID).build());
         store.jobFinished(compactionJobFinished(job2, defaultSummary()).taskId(DEFAULT_TASK_ID).build());
+        store.jobCommitted(compactionJobCommitted(job2, defaultCommitTime()).taskId(DEFAULT_TASK_ID).build());
```

Comment:
It looks like a finished but uncommitted job should be included in the results? Can we add a test case for that please? Maybe this test name could be specific that the job has been committed?

Reply:
I've decided to go with the naming scheme `uncommitted` if a job has finished but has not committed yet, and `finished` if a job has finished and committed. Do you think this is clear enough?

Reply:
That seems okay to me.